ui/src/main/java/org/pentaho/mantle/client/dialogs/scheduling/NewScheduleDialog.java

```diff
@@ -169,7 +172,13 @@ public void setScheduleName( String scheduleName ) {
   }
 
   public String getScheduleLocation() {
-    return scheduleLocationTextBox.getText();
+    return partiallyEncodedScheduleLocationPath;
+  }
+  
+  public static void setScheduleLocation( String path ){
+    partiallyEncodedScheduleLocationPath = path;
+    // TODO BACKLOG-41091: Decoding would ideally be coming from the back-end (see also, BACKLOG-41089, BACKLOG-41229)
+    scheduleLocationTextBox.setText( GenericFileNameUtils.urlDecodePvfsFilePath( path ) );
```

Comment:
The output location text box shows a generic file **path**. Paths of VFS files have the `%` character encoded, and the user should see the path unaltered. Not only that, but if, later, this path is further URL-encoded, to be integrated in the path of a URL, or the query string of a URL, or a URL-encoded HTTP body, the `%` character will be encoded twice. Reaching the server, the double encoding gets undone and the original, desired encoding of just the `%` character is restored to the path string. If this does not happen, the path fails to be parsed server-side by the Apache VFS system.
See related story and PR for PDI File Open/Save dialog: https://github.com/pentaho/pentaho-kettle/pull/9450.

The above use case is not the same as that of this story: a label in a folder/file tree/list node, where a _single segment_ of the path is displayed, in which case, individual segments are displayed non-encoded.

If I am right, then no change is needed in this class.